proposals/simd/SIMD.md

```diff
@@ -226,6 +230,11 @@ def S.shuffle(a, b, s):
     return result
 ```
 
+#### Variable permutation rule
+* `v8x16.shuffleVar(a: v128, b: v128, s: v128) -> v128`
```

Comment:
Small nit: wasm instructions usually are snake_case, so this would be `shuffle_var`.

Reply:
fixed